packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItem.jsx

```diff
@@ -153,6 +154,8 @@ class CollapsibleMenuViewItem extends React.Component {
               isReversed={isReversed}
               isDisabled={isDisabled}
               isIconOnly={isIconOnly}
+              isSelectable={isSelectable}
+              isSelected={isSelectedButton}
```

Comment:
```suggestion
```
This button is not a item, it is a anchor button that opens up a menu.

Reply:
https://github.com/cerner/terra-framework/pull/1902/commits/a9820cd176ac0301b8dcdacd01987d8f910511ab Added